internal/provider/registry/registry_test.go

```diff
@@ -25,9 +27,19 @@ var (
 const dummy = "dummy"
 const invalidInventory = "{\"system_vendor\": \"invalid\"}"
 
+const ovirtFqdn = "ovirt.example.com"
+const ovirtInsecure = true
```

Comment:
Please don't propagate the use of insecure. It's a very bad practice and is copied all over projects. Please use proper certificate verification, even in examples.